CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Draft
+- Style optimized checkout new checklist radio buttons [#dada](https://yadayada)
```

Comment:
And update the link here please :)

Reply:
To many screenshots, I forgot about it 😂